cla-backend-go/main.go

```diff
@@ -19,8 +25,25 @@ var (
 
 	// build date
 	buildDate string
+
+	configFile = ""
 )
 
+func init() {
+	stage := viper.GetString("STAGE")
+
+	awsSession, err := ini.GetAWSSession()
+	if err != nil {
+		log.Panicf("Unable to load AWS session - Error: %v", err)
+	}
+
+	configFile, err := config.LoadConfig(configFile, awsSession, stage)
```

Comment:
I think now we are loading the config twice.  Can we be smart and cache the results the second time this is called?

Reply:
Fixed loading config twice issue.